src/components/shared/chip/Chip.jsx

```diff
@@ -0,0 +1,31 @@
+import { Chip } from "@mui/material/";
+import { alpha, styled } from "@mui/material/styles";
+
+const StyledChip = styled(Chip, {
```

Comment:
Am realizat ca componenta data este defapt un Chip si nu Badge.

Reply:
ma indus in eroare asocierea  de aia am numit componenta badges

Reply:
nu mi dau seama ce inseamna importul ala alpha

Reply:
Alpha e un mecanism din Mui, analogia la `rgba()`

Reply:
Avantajul e ca ii putem pasa si valori hex, cum ar fi `background: alpha("#3754DB", 0.1),` unde prima valooare e culoarea, iar a 2-a opacitateal.